pos-print/src/main/java/com/clouway/pos/print/adapter/http/ReportService.kt

```diff
@@ -58,17 +57,24 @@ class ReportService(private var factory: PrinterFactory) {
     return Reply.with("Report completed").`as`(Json::class.java).ok()
   }
 
-  private fun adapt(state: String): RegisterState {
+  private fun adaptToState(state: String): RegisterState {
     if (state.equals("CLEAR")) {
       return RegisterState.CLEAR
     }
     return RegisterState.KEEP
   }
 
-  private fun adaptToDate(date: String): LocalDate {
-    return LocalDate.parse(date, DateTimeFormatter.ISO_DATE_TIME)
+  private fun adaptToType(type: String): PeriodType {
+    if (type.equals("SHORT")) {
+      return PeriodType.SHORT
+    }
+    return PeriodType.EXTENDED
+  }
+
+  private fun adaptToDate(date: String): LocalDateTime {
+    return LocalDateTime.parse(date, DateTimeFormatter.ISO_DATE_TIME)
```

Comment:
I think thain the tests it will be better if use directly the correct input or output

```java
 val reply = reportService.printPeriodReport(request.with("{\"sourceIp\":\"::sourceIp::\",\"from\":\"2017-01-01T00:00:00.000Z\",\"to\":\"2017-01-02T00:00:00.000Z\",\"periodType\":\"SHORT\"))
```
or
```java
 val reply = reportService.printPeriodReport(JsonBuilder.aNewJson()
                    .add("sourceIp", "::sourceIp::")
                    .add("from", "2017-01-01T00:00:00.000Z")
                    .add("to", "2017-01-02T00:00:00.000Z")
                    .add("periodType", "SHORT")
                    .build())
```